### PR TITLE
Symfony3 support

### PR DESCRIPTION
**Pull request by tgalopin**

### CODE REVIEW

**Comment by ubermuda** [comment]
@tgalopin Hi! Are you still working on this PR?

File: .travis.yml [hunk 1]
```diff
@@ -26,5 +26,4 @@ notifications:
 matrix:
   allow_failures:
     - env: SYMFONY_VERSION=dev-master
```
Comment (Ener-Getick) [inline .travis.yml]: I don't see the interest of keeping this in the allow failures section as almost no one look at them.
I prefer that the tests break and that we know that we will have to change something.

**Comment by Ener-Getick** [comment]
There is still a lot of work here as many part of this bundle are not covered by tests.

**Comment by tgalopin** [comment]
Yes, this is very complicated, as the tests are not ready. I'll have to find time to work on the library in order to build something stable for Symfony3. I'm sorry of this :/ .

**Comment by Ener-Getick** [comment]
No problem @tgalopin, you're already doing an awesome job ☺
Maybe the easiest solution is to add tests in smaller PRs and then merge them here to check if they work on symfony 3.0.

**Comment by tgalopin** [comment]
@Ener-Getick if we want to adapt this version for Symfony3, it would the best way indeed but I think we should focus the work on the next version that will use the library instead. This version is officially supported for fixes, not for new features (and Symfony3 support is a new feature IMO).

**Comment by ubermuda** [comment]
Yes, no problem, I just wanted to know what's up :)

FWIW, I started to port stuff to 3.0 so I could migrate one of our project. It's certainly not "the best way to do things", but feel free to have a look and re-use anything you think could help. I've got everything I need working – which is basic features like creating thread and replying. I don't know much about this bundle, so I have no idea how far I've gone in the process of porting the entire thing.

* Branch: https://github.com/ubermuda/FOSMessageBundle/tree/symfony3
* Diff: https://github.com/FriendsOfSymfony/FOSMessageBundle/compare/symfony3...ubermuda:symfony3

**Comment by tgalopin** [comment]
@ubermuda thanks for the work, I'll check that asap :) !

**Comment by Gemorroj** [comment]
any news?

**Comment by dmatora** [comment]
When i am trying to add
```
fos_message:
    db_driver: orm
    thread_class: MyBundle\Model\Thread
    message_class: MyBundle\Model\Message
```
to config.yml and `new FOS\MessageBundle\FOSMessageBundle(),` to AppKernel.php
i get
```
InvalidArgumentException in XmlFileLoader.php line 263: Unable to parse file "/symfony/vendor/friendsofsymfony/message-bundle/FOS/MessageBundle/DependencyInjection/../Resources/config/config.xml".
```
and
```
InvalidArgumentException in XmlUtils.php line 96: [ERROR 1866] Element '{http://symfony.com/schema/dic/services}service', attribute 'scope': The attribute 'scope' is not allowed. (in /symfony/web/ - line 61, column 0)
```

**Comment by dmatora** [comment]
Oh, just realized @ubermuda has fixed it in his fork.
Thank you! 👍

**Comment by tgalopin** [comment]
I'll test this tonight, sorry for the delay. Thanks for your work @ubermuda !

**Comment by dmatora** [comment]
Added small fix

**Comment by ownede** [comment]
Just meregd @ubermuda's code into my fork.
Creating a new thread, replying, views and forms (outbox/inbox/deleted/new message/reply) seems to work great. Need some more time for testing.

Branch: https://github.com/ownede/FOSMessageBundle/tree/symfony3

**Comment by Ener-Getick** [comment]
Closed in favor of #283

**Pull request closed without merging**